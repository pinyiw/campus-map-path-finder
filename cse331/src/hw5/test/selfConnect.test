# Create a graph
CreateGraph g

# Add nodes
AddNode g a
AddNode g b
AddNode g c

# Add edges
AddEdge g a a e1
AddEdge g b b e2
AddEdge g c c e3

#Print the outgoing edges from a, b and c
ListChildren g a
ListChildren g b
ListChildren g c