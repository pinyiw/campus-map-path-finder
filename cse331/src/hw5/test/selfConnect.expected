# Create a graph
created graph g

# Add nodes
added node a to g
added node b to g
added node c to g

# Add edges
added edge e1 from a to a in g
added edge e2 from b to b in g
added edge e3 from c to c in g

#Print the outgoing edges from a, b and c
the children of a in g are: a(e1)
the children of b in g are: b(e2)
the children of c in g are: c(e3)